test/integration/scale/scale_test.go

```diff
@@ -133,6 +140,17 @@ func TestScaleSubresources(t *testing.T) {
 	if _, err := clientSet.AppsV1().StatefulSets("default").Create(context.TODO(), ssStub, metav1.CreateOptions{}); err != nil {
 		t.Fatal(err)
 	}
+	if _, err := clientSet.AppsV1().DaemonSets("default").Create(context.TODO(), daemonsetStub, metav1.CreateOptions{}); err != nil {
+		t.Fatal(err)
+	}
+
+	kindRestMapping := map[string]interface{}{
+		"replicationcontrollers": &repContStorage.ScaleREST{},
+		"replicasets":            &repSetStorage.ScaleREST{},
+		"deployments":            &deployStorage.ScaleREST{},
+		"statefulsets":           &statSetStorage.ScaleREST{},
+		"daemonsets":             &daemonStorage.ScaleREST{},
+	}
```

Comment:
same comment about using discovery rather than linking to REST implementations here